### ISSUE TITLE
Links in digital PDFs should be visually highlighted

### ISSUE THREAD

**Issue by greatislander** [pressbooks/pressbooks#1306]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing this issue with one of our built-in root or book themes?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

In print PDF, links are removed. However in digital PDF, they are present but not distinguished from surrounding content.

### Steps to Reproduce

1. Add a link to the content of a chapter.
2. Export to digital PDF.
3. Find the link.

**Expected behavior:** It is underlined and (optionally) a different colour.

**Actual behavior:** It is identical to the surrounding content.

### System Information

Pressbooks: 5.5.0-dev
McLuhan: 2.5.0-dev

**Comment by greatislander** [comment]
Blocked by #1437.
